### ISSUE TITLE
BlenderBim:  IfcOpeningElement "destructive" to the mesh after file save or geometry representation updated

### ISSUE THREAD

**Issue by Tudjack** [IfcOpenShell/IfcOpenShell#3596]
Is there a way after applying a void opening to an object, that void could be modified after file is being saved without the object's mesh to take the shape of the void? Basically a non-destructive approach.

**Comment by Moult** [comment]
Opening elements are already non-destructive. Just toggle opening modes (Alt-O) with the BIM Tool activated and you can edit openings.

**Comment by Tudjack** [comment]
I am sorry, but there must be a bug on my end. I know how to use the toggle for opening modes with the BIM tool, but still, my mesh is getting the permanent modification from the void.
If I move the void around or change it, the former cut shape remains in the mesh. 
This is happening after I save the file. It seems like an application of a boolean modifier to a mesh.
I created the mesh with native Blender and classified it manually, I did not use any type class from any library. Could that be a cause?
